Office 365/message-trace/ingest/parser.yml

```
@@ -35,12 +22,13 @@ stages:
   extract_fields:
     actions:
       - set:
+          event.kind: "event"
+          event.category: "email"
+          event.type: "info"
+          '@timestamp': "{{parse_received_date.datetime}}"
```

It seems that the stage `parse_received_date` publish its result in the field `date`
```suggestion
          '@timestamp': "{{parse_received_date.date}}"
```